graphql_compiler/compiler/emit_sql.py

```diff
@@ -743,7 +743,7 @@ def __init__(self, sql_schema_info: SQLAlchemySchemaInfo, ir: IrAndMetadata):
         self._relocate(ir.query_metadata_table.root_location)
 
         # Mapping aliases to the column used to join into them.
-        self._came_from: Dict[Alias, Column] = {}
+        self._came_from: Dict[Union[Alias, ColumnRouter], Column] = {}
```

Comment:
The mention of `aliases` in the docstring is outdated. So is the entire `_aliases` field, since it doesn't only contain aliases anymore. I'm working on fixing this in the next PR, and I created an issue https://github.com/kensho-technologies/graphql-compiler/issues/882